dev/tests/integration/testsuite/Magento/Shipping/Block/Adminhtml/Order/AddToPackageTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Shipping\Block\Adminhtml\Order;
+
+use Magento\Backend\Block\Template;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Framework\Registry;
+use Magento\Sales\Api\Data\OrderInterfaceFactory;
+use Magento\Sales\Api\Data\ShipmentTrackInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class verifies packaging popup.
+ *
+ * @magentoAppArea adminhtml
+ */
+class AddToPackageTest extends TestCase
+{
+    /** @var ObjectManagerInterface */
+    private $objectManager;
+
+    /** @var Registry */
+    private $registry;
+    /**
+     * @var OrderInterfaceFactory|mixed
+     */
+    private $orderFactory;
+
+    protected function setUp(): void
+    {
+        $this->objectManager = Bootstrap::getObjectManager();
+        $this->registry = $this->objectManager->get(Registry::class);
+        $this->orderFactory = $this->objectManager->get(OrderInterfaceFactory::class);
+    }
+
+    /**
+     * Test that Packaging popup renders
+     *
+     * @magentoDataFixture Magento/GraphQl/Sales/_files/customer_order_with_ups_shipping.php
```

Comment:
Can we use a fixture from the Sales module instead? It sounds more appropriate. Like this one `dev/tests/integration/testsuite/Magento/Sales/_files/order_with_shipping_and_invoice.php`

Reply:
We may even use the fixture from the `Shipping` module `dev/tests/integration/testsuite/Magento/Shipping/_files/track.php` it also creates an order with shipment.